A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js

```diff
@@ -90,27 +100,46 @@ var A320_Neo_LowerECAM_APU;
             this.APUFrequency.textContent = SimVar.GetSimVarValue("L:APU_GEN_FREQ","Hertz");
             this.APUFrequency.setAttribute("class", "APUGenParamValue");
 
-            //AVAIL indication & bleed pressure
-            if (APUPctRPM > 95) {
-                this.APUAvail.setAttribute("visibility", "visible");
+            const adirsAligned = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Number") === 2;
+            const apuGenActive = SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") === 1;
+            const externalPowerOn = SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0;
+
+            // AVAIL indication & bleed pressure
+            const doUpdateAvailAndBleed =
+                   APUPctRPM !== this.previousState.APUPctRPM
+                || adirsAligned !== this.previousState.adirsAligned
+                || apuGenActive !== this.previousState.apuGenActive
+                || externalPowerOn !== this.previousState.externalPowerOn;
+
+            if (doUpdateAvailAndBleed) {
+                this.previousState.APUPctRPM = APUPctRPM;
+                this.previousState.adirsAligned = adirsAligned;
+                this.previousState.apuGenActive = apuGenActive;
+                this.previousState.externalPowerOn = apuGenActive;
+
+                if (APUPctRPM > 95) {
+                    this.APUAvail.setAttribute("visibility", "visible");
+
+                    if (apuGenActive && externalPowerOn) {
+                        this.APUGenAvailArrow.setAttribute("visibility", "visible");
+                    } else {
+                        this.APUGenAvailArrow.setAttribute("visibility", "hidden");
+                    }
+
+                    this.APUBleedPressure.textContent = SimVar.GetSimVarValue("L:APU_BLEED_PRESSURE","PSI");
+                    this.APUBleedPressure.setAttribute("class", "APUGenParamValue");
+                }
 
-                if (SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") == 1 && SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0) {
-                    this.APUGenAvailArrow.setAttribute("visibility", "visible");
-                } else {
+                if (APUPctRPM < 95 || !adirsAligned) {
```

Comment:
If it is a float is probably doesn't matter but `APUPctRPM <= 95` might be better considering the other bound is `APUPctRPM > 95`